library/core/src/ptr/mut_ptr.rs

```diff
@@ -2359,8 +2370,192 @@ impl<T: ?Sized> PartialOrd for *mut T {
 
 #[cfg(kani)]
 #[unstable(feature = "kani", issue = "none")]
-mod verify {
+pub mod verify {
```

Comment:
Curious why this is necessary?

Reply:
Sorry! It isn't necessary. I removed it.